pages/draft/2020-12/release-notes.md

```diff
@@ -156,8 +162,8 @@ Here's how you would covert a schema using `$recursiveRef` to use `$dynamicRef`.
   <tr>
     <td>
       ```json
-        // tree schema, extensible
         {
+          "comment": "tree schema extensible",
```

Comment:
This comment should be listed as a title for the code block rather than being incorporated into the schema.